app/0.4.x/getting-started/introduction.md

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
same as before. Any reason why these intro pages are being treated as new?

Reply:
Really strange, not sure why its doing that. Same file, nothing changed on it